pcsx2/gui/Dialogs/AboutBoxDialog.cpp

```diff
@@ -54,62 +54,47 @@ Dialogs::AboutBoxDialog::AboutBoxDialog( wxWindow* parent )
+		L"%s: black_wd, Belmont, BGome, _Demo_, Dreamtime, "
+		L"F|RES, Jake.Stine, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos",
+		_("Plugin Specialists"), _("Special thanks to"));
 
-	// this sizer holds text of the contributors/testers, and a ps2 image!
-	wxBoxSizer& ContribSizer = *new wxBoxSizer( wxHORIZONTAL );
+	wxFlexGridSizer& boxesContainer = *new wxFlexGridSizer(2, 0, StdPadding);
+	boxesContainer.AddGrowableCol(0, 1);
+	boxesContainer.AddGrowableCol(1, 1);
 
-	wxStaticBoxSizer& aboutUs	= *new wxStaticBoxSizer( wxVERTICAL, this );
-	wxStaticBoxSizer& contribs	= *new wxStaticBoxSizer( wxVERTICAL, this );
+	wxStaticBoxSizer& developsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
+	wxStaticBoxSizer& contribsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
 
-	pxStaticText& label_auth	= Text( LabelAuthors ).SetMinWidth(240);
-	pxStaticText& label_greets	= Text( LabelGreets ).SetMinWidth(200);
+	pxStaticText& developsText = Text(developsString).SetMinWidth(220);
+	pxStaticText& contribsText = Text(contribsString).SetMinWidth(220);
 
-	aboutUs		+= Heading(_("Developers")).Bold()	| StdExpand();
-	aboutUs		+= label_auth						| StdExpand();
-	contribs	+= Heading(_("Contributors")).Bold()	| StdExpand();
-	contribs	+= label_greets						| StdExpand();
+	developsBox += Heading(_("Developers")).Bold() | StdExpand();
+	developsBox += developsText | StdExpand();
+	contribsBox += Heading(_("Contributors")).Bold() | StdExpand();
+	contribsBox += contribsText | StdExpand();
 
-	AuthLogoSizer	+= aboutUs		| StdExpand();
-	AuthLogoSizer	+= contribs		| StdExpand();
+	boxesContainer += developsBox | StdExpand();
+	boxesContainer += contribsBox | StdExpand();
 
-	ContribSizer	+= pxStretchSpacer( 1 );
-	ContribSizer	+= m_bitmap_dualshock	| StdSpace();
-	ContribSizer	+= pxStretchSpacer( 1 );
+	// Main layout
+	*this += m_bitmap_logo | StdCenter();
+	*this += Text(_("PlayStation 2 Emulator"));
 
-	// Main (top-level) layout
+	*this += boxesContainer;
 
-	*this	+= StdPadding;
-	*this	+= Text(wxGetApp().GetAppName()).Bold();
-	*this	+= Text(_("A Playstation 2 Emulator"));
-	*this	+= AuthLogoSizer						| StdExpand();
-
-	*this	+= new wxHyperlinkCtrl( this, wxID_ANY,
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
 		_("PCSX2 Official Website and Forums"), L"http://www.pcsx2.net"
-	) | pxProportion(1).Center().Border( wxALL, 3 );
+		) | pxProportion(1).Center().Border(wxALL, 3);
 
-	*this	+= new wxHyperlinkCtrl( this, wxID_ANY,
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
 		_("PCSX2 Official Git Repository at GitHub"), L"https://github.com/PCSX2/pcsx2"
-	) | pxProportion(1).Center().Border( wxALL, 3 );
-
-	*this	+= ContribSizer											| StdExpand();
-	*this	+= new wxButton( this, wxID_OK, _("I've seen enough"))	| StdCenter();
+		) | pxProportion(1).Center().Border(wxALL, 3);
 
-	int bestHeight = GetBestSize().GetHeight();
-	if( bestHeight < 400 ) bestHeight = 400;
-	SetMinHeight( bestHeight );
+	SetMinHeight(GetBestSize().GetHeight());
```

Comment:
This line alters the vertical spacing of the hyperlinks for some reason. For the other dialogs I used `SetSizerAndFit(GetSizer());` for automatic sizing. There's probably a better way but it avoids a dialog size bug on Linux.